build.sbt

```diff
@@ -1,6 +1,10 @@
 lazy val root = (project in file(".")).
-  aggregate(contrib, mqtt)
+  aggregate(contrib, mqtt, amqp).
+  enablePlugins(GitVersioning)
 
 lazy val contrib = project
 lazy val mqtt = project
 lazy val amqp = project
+
+git.useGitDescribe := true
```

Comment:
Please add such info to the README, I'll totally screw up the order or something ;)